pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -34,11 +34,13 @@
     "version": "1.8.0"
   },
   "alicloud": {
-    "owner": "terraform-providers",
+    "owner": "aliyun",
+    "provider-source-address": "registry.terraform.io/aliyun/alicloud",
     "repo": "terraform-provider-alicloud",
-    "rev": "v1.86.0",
-    "sha256": "1hbv9ah7fd173sapwgsbg7790piwxw9zx90wfj5vz5b96ggbg28d",
-    "version": "1.86.0"
+    "rev": "v1.144.0",
+    "sha256": "14nphpz15p83n6fsvvrnaz96nb87wvb10ri21hlhlsm2579zcbqd",
+    "vendorSha256": null,
```

Comment:
This one might need an actual value. You can insert a fake checksum and than trigger a build to get the actual one.

Reply:
```
unpacking source archive /private/tmp/nix-build-source.drv-0/v1.144.0.tar.gz
building '/nix/store/jjdlv7l734fl65d95zb5yd6k95glkcda-terraform-provider-alicloud-1.144.0-go-modules.drv'...
unpacking sources
unpacking source archive /nix/store/58hjmd9flaq2qwql48a0gjzxar92pj70-source
source root is source
patching sources
updateAutotoolsGnuConfigScriptsPhase
configuring
building
vendor folder exists, please set 'vendorSha256 = null;' in your expression
error: builder for '/nix/store/jjdlv7l734fl65d95zb5yd6k95glkcda-terraform-provider-alicloud-1.144.0-go-modules.drv' failed with exit code 10;
       last 8 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/58hjmd9flaq2qwql48a0gjzxar92pj70-source
       > source root is source
       > patching sources
       > updateAutotoolsGnuConfigScriptsPhase
       > configuring
       > building
       > vendor folder exists, please set 'vendorSha256 = null;' in your expression
       For full logs, run 'nix log /nix/store/jjdlv7l734fl65d95zb5yd6k95glkcda-terraform-provider-alicloud-1.144.0-go-modules.drv'.

```

Reply:
Ok. Than the go.mod needs an update.